extras/travis_build.sh

```diff
@@ -168,25 +168,15 @@ if [ "$xSTATIC_ANALYSIS" = "1" ]; then
        echo "MSGBLD0080: PULL REQUEST: Retrieving changed files using GitHub API."
        file_names=`curl "https://api.github.com/repos/$TRAVIS_REPO_SLUG/pulls/$TRAVIS_PULL_REQUEST/files" | jq '.[] | .filename' | tr '\n' ' ' | tr '"' ' '`
     else
-       echo "MSGBLD0090: Retrieving changed files using git diff."    
+       echo "MSGBLD0090: Retrieving changed files using git diff."
        file_names=`(git diff --name-only $TRAVIS_COMMIT_RANGE || echo "") | tr '\n' ' '`
     fi
-
-    printf '%s\n' "$file_names" | while IFS= read -r line
-     do
-       for single_file_name in $file_names
-       do
-         echo "MSGBLD0095: File changed: $single_file_name"
-       done
-     done
-    echo "MSGBLD0100: Retrieving changed files completed."
-    echo
-
+    #file_names=`find . -name "*.h" -o -name "*.c" -o -name "*.cc" -o -name "*.hpp" -o -name "*.cpp" -o -name "*.py"`
```

Comment:
Please remove the commented out lines.

Reply:
I added a comment explaining the purpose (check *all* files instead of just the ones that have changed); this might come in handy for testing but should not be the default on Travis CI due to excessive runtimes. Note that over the course of time, some files slipped through, i.e. they are in nest-master but fail the static check. Some of these I fixed in this PR (these are the seemlingly unrelated changes in the diff).